daemon/daemon_unix.go

```diff
@@ -1555,7 +1555,9 @@ func (daemon *Daemon) initCgroupsPath(path string) error {
 
 	// Recursively create cgroup to ensure that the system and all parent cgroups have values set
 	// for the period and runtime as this limits what the children can be set to.
-	daemon.initCgroupsPath(filepath.Dir(path))
+	if err := daemon.initCgroupsPath(filepath.Dir(path)); err != nil {
+		return err
```

Comment:
please wrap

Reply:
if not here, then in initCgroupsPath

Reply:
This is `initCgroupsPath`; it's recursing. The final error is "wrapped" (well, `fmt.Errorf` - we should change that) here; https://github.com/moby/moby/blob/3042254a87274ff5e9561f2da1a986a703dfc60f/daemon/oci_linux.go#L793-L795

Reply:
I'm talking about package boundaries like lines 1564, 1583 and 1586

Reply:
ping @thaJeztah